packages/@okta/vuepress-site/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@okta/vuepress-site",
-  "version": "1.0.0-alpha.3",
+  "version": "1.0.0",
```

Comment:
The `-alpha.3` version tag was confusing before. I changed the artifact tagging below in `ci-append-sha`